src/EnergyPlus/ZoneDehumidifier.cc

```diff
@@ -1121,7 +1118,7 @@ namespace ZoneDehumidifier {
             // Calculate and report condensation rate (how much water extracted from the air stream)
             // Volumetric flow of water in m3/s for water system interactions
 
-            AirInletNodeNum = state.dataZoneDehumidifier->ZoneDehumid(DehumidNum).AirInletNodeNum;
+            int AirInletNodeNum = state.dataZoneDehumidifier->ZoneDehumid(DehumidNum).AirInletNodeNum;
```

Comment:
[src/EnergyPlus/ZoneDehumidifier.cc:1108]:(style),[variableScope],The scope of the variable 'AirInletNodeNum' can be reduced.
[src/EnergyPlus/ZoneDehumidifier.cc:766]:(style),[unreadVariable],Variable 'AirOutletNodeNum' is assigned a value that is never used.